wrappers/python/xraydb/xraydb.py

```diff
@@ -430,8 +430,29 @@ def xray_lines(self, element, initial_level=None, excitation_energy=None):
                                                r.initial_level, r.final_level)
         return out
 
+    def xray_line_strengths(self, element):
+        """
+        return the absolute line strength in cm^2/gr for all available lines
+
+        arguments
+        ---------
+        element:  atomic number, atomic symbol for element
+
+        """
+        out = {}
+        for label, eline in self.xray_lines(element).items():
```

Comment:
I think you need to use excitation_energy in self.xray_lines, and also pass it as a function argument? This calculation depends on incident energy.